cmd/admin/key.go

```diff
@@ -26,9 +28,13 @@ import (
 type subcommandBlockKey struct {
 	parallelism uint
 	comment     string
-	privKey     string
-	spkiFile    string
-	certFile    string
+
```

Comment:
I added a blank line here to separate the input methods into a block, mirroring their appearance in the `Flags` function